setup.cfg

```diff
@@ -6,17 +6,31 @@ summary = Simple dataset to dataloader library for pytorch
 description-file =
     README.rst
 description-content-type = text/x-rst; charset=UTF-8
+home-page = https://github.com/aiwizo/pytorch-datastream
 project_urls =
-    Source Code = https://github.com/aiwizo/pytorch-datastream/
+    Source Code = https://github.com/aiwizo/pytorch-datastream
+    Bug Tracker = https://github.com/aiwizo/pytorch-datastream/issues
 license = Apache-2
+license_files = LICENSE
+platform = any
 classifier =
     Development Status :: 3 - Alpha
     Environment :: Other Environment
     License :: OSI Approved :: Apache Software License
     Operating System :: OS Independent
     Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Intended Audience :: Developers
+    Intended Audience :: Science/Research
+    Topic :: Scientific/Engineering
+    Topic :: Scientific/Engineering :: Artificial Intelligence
+    Topic :: Software Development
+    Topic :: Software Development :: Libraries
+    Topic :: Software Development :: Libraries :: Python Modules
 keywords =
-    pytorch
+    pytorch torch dataset dataloader machine learning
```

Comment:
"Machine learning" will sadly be two different tags and not a single one

Reply:
not super sure how the algorithm for keyword search works, but I think we are in the clear, except that people searching for machine will also have us popping up
do you know how it works?

Reply:
I'm having trouble finding documentation at all for this. You might be right